### ISSUE TITLE
Cannot custom search-bar wrapper background color

### ISSUE THREAD

**Issue by wuzhishuo** [ant-design/ant-design-mobile#1158]
### Version
1.0.8

### Environment
macOS Sierra 10.12, react-native: 0.42.3

### Reproduction link
[https://github.com/ant-design/antd-mobile-samples/tree/master/rn-custom-ui](https://github.com/ant-design/antd-mobile-samples/tree/master/rn-custom-ui)

### Steps to reproduce
I want to use the way from [https://github.com/ant-design/antd-mobile-samples/tree/master/rn-custom-ui](https://github.com/ant-design/antd-mobile-samples/tree/master/rn-custom-ui) to custom the UI. 
1. use the custom-rn-theme.js from rn-custom-ui sample
2. create myself theme.js file, and the code is:
```
module.exports = {
    color_link: 'red',
    border_color_base: '#f4bc1c',
    search_color_icon: '#f4bc1c',
    fill_tap: '#f4bc1c',
    input_color_icon: '#f4bc1c',
    link_button_fill_tap: '#f4bc1c',
    search_bar_fill: '#f4bc1c',
};

```
3. use node to run custom-rn-theme.js before start to run my react-native project

### What is expected?
The antd-mobile components should use the variables I defined to render the UI.

### What is actually happening?
* Other variables I defined works well except the variable search_bar_fill, the background of search-bar wrapper doesn't be set to '#f4bc1c'.
* Actually the background of search-bar wrapper is '#ddd', event not the color(#efeff4) defined in components/style/themes/default.tsx 
* the backgroundColor in /node_modules/antd-mobile/lib/search-bar/style/index.js file is hardcode in '#ddd', and this file seems not not come from the source code of version 1.0.8 but version 1.0.7. Here is the code:
```
'use strict';Object.defineProperty(exports,"__esModule",{value:true});var _default=require('../../style/themes/default');var _default2=_interopRequireDefault(_default);
var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{"default":obj};}exports["default"]=
_reactNative.StyleSheet.create({
input:{
borderRadius:_default2["default"].radius_md,
backgroundColor:'#fff',
borderColor:_default2["default"].border_color_base,
borderWidth:_default2["default"].border_width_sm,
alignSelf:'stretch',
height:_default2["default"].search_bar_input_height,
color:_default2["default"].color_text_base,
fontSize:_default2["default"].font_size_base,
paddingLeft:_default2["default"].h_spacing_lg _default2["default"].icon_size_xxs _default2["default"].h_spacing_sm,
paddingRight:_default2["default"].h_spacing_lg _default2["default"].icon_size_xxs _default2["default"].h_spacing_sm,
flex:1,
paddingTop:0,
paddingBottom:0},

wrapper:{
backgroundColor:'#ddd',
paddingTop:8,
paddingBottom:8,
paddingLeft:_default2["default"].h_spacing_md,
paddingRight:_default2["default"].h_spacing_md,
flexDirection:'row'},

cancelTextContainer:{
height:_default2["default"].search_bar_input_height,
justifyContent:'center',
alignItems:'center'},

cancelText:{
fontSize:_default2["default"].link_button_font_size,
color:_default2["default"].color_link,
paddingLeft:_default2["default"].h_spacing_lg},

search:{
tintColor:_default2["default"].input_color_icon,
position:'absolute',
left:_default2["default"].h_spacing_md 8,
top:(_default2["default"].search_bar_input_height-_default2["default"].icon_size_xxs)/2 8,
width:_default2["default"].icon_size_xxs,
height:_default2["default"].icon_size_xxs}});module.exports=exports['default'];

```

**Comment by paranoidjk** [comment]
https://github.com/ant-design/ant-design-mobile/blob/44472702003cfcfe115ff80027a4d369c8ba7137/components/style/themes/default.tsx#L157

Whats your problem actually is ?  `search_bar_fill` and `input_color_icon` is totally different usage.

**Comment by wuzhishuo** [comment]
Sorry, please forget the `input_color_icon`. My problem is cannot use `search_bar_fill` to custom the background color of search-bar. The background I mean here is the padding around the TextInput.

**Comment by silentcloud** [comment]
@wuzhishuo  https://github.com/ant-design/ant-design-mobile/commit/44472702003cfcfe115ff80027a4d369c8ba7137#diff-d3415ef881a8cdb8aa6b590b94b43267

This will be fixed in `1.0.9`
